app.ts

```diff
@@ -0,0 +1,89 @@
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+import * as fs from 'fs';
+
+const host: string = '127.0.0.1';
+const port: number = 3000;
+const path: string = './newFile.txt';
+
+createServer((request: IncomingMessage, response: ServerResponse) => {
+    switch (request.method) {
+        case 'GET': {
+            fs.readFile(path, 'utf-8', (error, data) => {
```

Comment:
Move encoding to the constants file.
Wrong formatting. Use 2 spaces instead of 4.